cassandra_nodetool/assets/configuration/spec.yaml

```diff
@@ -13,10 +13,9 @@ files:
           The nodetool installation also sets some environment variables (for example, `CASSANDRA_HOME`
           and `DSCINSTALLDIR`), but sets them as variables only for the user doing the nodetool installation.
           You should set those environment variables as system-wide variables.
-        required: true
         value:
           type: string
-          example: nodetool
+          example: /usr/bin/nodetool
       - template: init_config/default
   - template: instances
```

Comment:
Now that we're at this `instances` section is missing the `nodetool` definition.
In code it looks like this:

```
 default_nodetool_cmd = init_config.get("nodetool", "/usr/bin/nodetool")
 nodetool_cmd = self.instance.get('nodetool', default_nodetool_cmd).split()
```